backend/src/test/resources/rbac-examples/rbac_example_full_access.json

```diff
@@ -15,6 +15,10 @@
       "permission": "notifications:*:*",
       "resourceDefinitions": []
     },
+    {
+      "permission": "events:*:*",
```

Comment:
We probably don't need this one, the first is the application (notifications). We are not creating a new `application`, only a new `resource`

Reply:
i just tested it. When I remove this, the EventServiceTest.testAllQueryParams() fails.

Reply:
It goes with the other comment :)

Reply:
ok, then it's resolved I guess :)